packages/app/src/common/api/fetch.ts

```diff
@@ -1,9 +1,16 @@
+import packageJson from '../../../package.json';
+
 export const withApiServer = (apiServer: string) => (path?: string) =>
   path ? apiServer + path : apiServer;
 
+const defaultHeaders = new Headers([
+  ['x-stx-extension-version', packageJson.version]
```

Comment:
How can we ensure these are the same? https://github.com/blockstack/ux/blob/release/canary/packages/app/src/manifest.json#L4

Reply:
https://github.com/blockstack/ux/blob/release/canary/packages/app/webpack.config.js#L238
https://github.com/blockstack/ux/blob/release/canary/packages/app/webpack.config.js#L245
We should use this function to generate the version in the header: https://github.com/blockstack/ux/blob/release/canary/packages/app/webpack.config.js#L41

Reply:
I could import version from there, instead?

Reply:
No I think it has to be a replace kind of thing. In the webpack config, it's generated on build and then with CopyWebpackPlugin, it's replacing the string with a random number appended to the package.json version

Reply:
we should use `process.env.VERSION`, it's already added!

Reply:
it comes out undefined